### PR TITLE
Add basic setup scripts, handle some drake updates

### PR DESCRIPTION
**Pull request by EricCousineau-TRI**
Not sure if it's useful, but this consolidates + localizes the Blender modifications (and just uses `ensurepip + pip` to install the packages).

Also did some Drake updates - doesn't fix all deprecations, though...

(We've got goals for photorealism, definitely gonna use this!)

---
This change is [Reviewable](https://reviewable.io/reviews/gizatt/blender_server/7)

### CODE REVIEW

**Comment by gizatt** [comment]
This is amazing! Thanks so much! I'd like to try it out myself, but I'm busy today and tomorrow -- is it OK if this sits for a few days?

**Comment by EricCousineau-TRI** [comment]
Certainly, take your time!

File: data/ycb_downloader.py [hunk 1]
```diff
@@ -0,0 +1,111 @@
...
+    html = response.read()
+    objects = json.loads(html)
+    return objects["objects"]
+
+def download_file(url, filename):
+    u = urllib2.urlopen(url)
+    f = open(filename, 'wb')
+    meta = u.info()
+    file_size = int(meta.getheaders("Content-Length")[0])
+    print "Downloading: %s (%s MB)" % (filename, file_size/1000000.0)
```
Comment (gizatt) [inline data/ycb_downloader.py]: I think this might be the only thing keeping it from running with python 3?

File: data/ycb_downloader.py [hunk 2]
```diff
@@ -0,0 +1,111 @@
+#!/usr/bin/env python2
```
Comment (gizatt) [inline data/ycb_downloader.py]: Nit: could this be Python 3 compatible?

**Review by gizatt — COMMENTED** [review]
Thanks a bunch, again! The setup script works great for me -- will make life much easier. One tiny request concerning a Python 2 script here, but if it's a bother I'm happy to merge as-is.

**Comment by gizatt** [comment]
BTW, if I keep pushing features to master (I'll try to make sure they usually work :)), does that break your workflow too much? I'll eventually want to merge in things like the label image branch, and I may extend the amount of data / domain randomization available if I start using this to actually train systems.

File: data/ycb_downloader.py (hunk 1 above)
Comment (EricCousineau-TRI) [reply data/ycb_downloader.py]: Done.

File: data/ycb_downloader.py (hunk 2 above)
Comment (EricCousineau-TRI) [reply data/ycb_downloader.py]: Done.

**Comment by EricCousineau-TRI** [comment]
I don't think that should break my workflow; if the API changes when I update, I can always upgrade or stay at a pin, so I should be good. (If I have my own major updates, I'll try to upstream them, of course!)
Thanks for asking!

**Comment by gizatt** [comment]
Excellent, thanks!

**Pull request merged**